lib/shopify-cli/commands/authenticate.rb

```diff
@@ -0,0 +1,23 @@
+require 'shopify_cli'
+
+module ShopifyCli
+  module Commands
+    class Authenticate < ShopifyCli::Command
+      def call(args, _name)
+        spin_group = CLI::UI::SpinGroup.new
+        spin_group.add("Requesting access token...") do |spinner|
+          ShopifyCli::Tasks::AuthenticateShopify.call(@ctx)
```

Comment:
If an access token is already present and is valid, what happens?

Reply:
It reprints with the same access token actually! I tested it a few times